myapp/docker-compose.yml

```diff
@@ -13,14 +13,26 @@ services:
 
   api:
     build: .
-    command: bash -c "bundle exec rails db:create && bundle exec rails db:migrate && rm -f tmp/pids/server.pid && bundle exec rails s -p 3000 -b '0.0.0.0'"
+    command: bash -c "bundle install && yarn install && bundle exec rails webpacker:install && bundle exec rails db:create && bundle exec rails db:migrate && rm -f tmp/pids/server.pid && bundle exec rails s -p 3000 -b '0.0.0.0'"
     volumes:
       - .:/myapp
       - bundle-install:/usr/local/bundle:delegated
     ports:
       - "3001:3000"
     links:
       - db
+    depends_on:
+      - chrome # <-- Link to the chrome container
+    environment: # <-- environment configs for database.yml
+      DB_USER: root
+      DB_NAME: myapp
+      DB_PASSWORD: password
```

Comment:
[optional]
Maybe add comment stating that "change these if you want" on the field DB_NAME and DB_PASSWORD.